{"uuid": "37f17259-d691-45c4-be1c-d7e922a9d323", "children": ["71d85e61-fdbf-42f8-b5bb-b1dcc9c217e2"], "befores": [{"name": "remote", "status": "passed", "attachments": [{"source": "2f3072f7-3205-44e5-97e6-04b6eab4e31c-attachment.json", "type": "application/json"}], "start": 1604856513971, "stop": 1604856521922}], "afters": [{"name": "remote::fin", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/usr/local/lib/python3.6/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/app/conftest.py\", line 52, in fin\n    driver.quit()\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 698, in quit\n    self.execute(Command.QUIT)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"/usr/local/lib/python3.6/site-packages/selenium/webdriver/remote/remote_connection.py\", line 402, in _request\n    resp = http.request(method, url, body=body, headers=headers)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/request.py\", line 76, in request\n    method, url, fields=fields, headers=headers, **urlopen_kw\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/request.py\", line 97, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/poolmanager.py\", line 336, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 677, in urlopen\n    chunked=chunked,\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 426, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"/usr/local/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 421, in _make_request\n    httplib_response = conn.getresponse()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 1379, in getresponse\n    response.begin()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 311, in begin\n    version, status, reason = self._read_status()\n  File \"/usr/local/lib/python3.6/http/client.py\", line 272, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/usr/local/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1604856526256, "stop": 1604856565067}], "start": 1604856513971, "stop": 1604856565068}